backend/import/.test_data/output/0f09cea1-58d7-41c9-85cd-692bd84adf38.json

```diff
@@ -1 +1 @@
\ No newline at end of file
+{"uuid": "0f09cea1-58d7-41c9-85cd-692bd84adf38", "associated": [], "employees": [], "locations": [["Returadresse", "AdressePostRetur", "DAR", "Nr Bjertvej 2, Bjert, 7830 Vinderup"], ["Email", "EmailUnit", "EMAIL", "Skoler_og_b\u00f8rnehaver@holstebro.dk"], ["Henvendelsessted", "AdresseHenvendelsessted", "DAR", "Nr Bjertvej 2, Bjert, 7830 Vinderup"], ["Postadresse", "AddressMailUnit", "DAR", "Nr Bjertvej 2, Bjert, 7830 Vinderup"], ["EAN Nummer", "EAN", "EAN", "3646476505434"], ["P-nummer", "p-nummer", "PNUMBER", "7550807866"]], "departments": [["Haderup skole", "efe464ea-e955-424c-9ee1-1c392b1453bc"], ["Hjerm skole", "be2fb801-b1a3-46a1-aedf-30bebdacddd8"], ["Holstebro skole", "fccbcd61-317a-4f5b-a6c0-198fdfbb81af"], ["Skive skole", "d7db32d2-bcd4-4c42-ab55-39cf52da6746"], ["Struer skole", "d4e24465-a9ea-4528-b53d-3b892d33eab7"], ["Ulfborg skole", "ccdb652e-1bc0-4917-8b20-c36a8e6c17a2"], ["Vemb skole", "a2ad1a71-1841-45ce-9834-846f9d0d1905"], ["Vinderup skole", "2f648d17-441a-4646-87b4-ecc2bd4eb5e7"]], "name": "Skoler og b\u00f8rnehaver", "document": "{\"associated\": [], \"departments\": [[\"Haderup skole\", \"efe464ea-e955-424c-9ee1-1c392b1453bc\"], [\"Hjerm skole\", \"be2fb801-b1a3-46a1-aedf-30bebdacddd8\"], [\"Holstebro skole\", \"fccbcd61-317a-4f5b-a6c0-198fdfbb81af\"], [\"Skive skole\", \"d7db32d2-bcd4-4c42-ab55-39cf52da6746\"], [\"Struer skole\", \"d4e24465-a9ea-4528-b53d-3b892d33eab7\"], [\"Ulfborg skole\", \"ccdb652e-1bc0-4917-8b20-c36a8e6c17a2\"], [\"Vemb skole\", \"a2ad1a71-1841-45ce-9834-846f9d0d1905\"], [\"Vinderup skole\", \"2f648d17-441a-4646-87b4-ecc2bd4eb5e7\"]], \"employees\": [], \"locations\": [[\"Returadresse\", \"AdressePostRetur\", \"DAR\", \"Nr Bjertvej 2, Bjert, 7830 Vinderup\"], [\"Email\", \"EmailUnit\", \"EMAIL\", \"Skoler_og_b\\u00f8rnehaver@holstebro.dk\"], [\"Henvendelsessted\", \"AdresseHenvendelsessted\", \"DAR\", \"Nr Bjertvej 2, Bjert, 7830 Vinderup\"], [\"Postadresse\", \"AddressMailUnit\", \"DAR\", \"Nr Bjertvej 2, Bjert, 7830 Vinderup\"], [\"EAN Nummer\", \"EAN\", \"EAN\", \"3646476505434\"], [\"P-nummer\", \"p-nummer\", \"PNUMBER\", \"7550807866\"]], \"managers\": [], \"name\": \"Skoler og b\\u00f8rnehaver\", \"parent\": \"39fb2929-c6c2-4a41-b14b-cdadf0d8ee2b\", \"root_uuid\": \"b1d6b1b6-d7d0-4e43-9d80-646395b7bfac\", \"uuid\": \"0f09cea1-58d7-41c9-85cd-692bd84adf38\"}", "parent": "39fb2929-c6c2-4a41-b14b-cdadf0d8ee2b", "managers": [], "root_uuid": "b1d6b1b6-d7d0-4e43-9d80-646395b7bfac"}
```

Comment:
Your line length is ever so slightly excessive, in my opinion 😇

Reply:
Will fix next time I revise the tests.